### PR TITLE
feat(tutorial): complete step 2 - Sao Paulo

### PR DESCRIPTION
**Pull request by Taciones**
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/content/LandingPage/LandingPage.js [hunk 1]
```diff
@@ -1,7 +1,104 @@
...
+                <a href="/">Getting started</a>
+              </BreadcrumbItem>
+            </Breadcrumb>
+            <h1 className="landing-page__heading">
+              Design &amp; build with Carbon
+            </h1>
+          </div>
+        </div>
+      </div>
+      <div className="bx--row landing-page__r3">
```
Comment (kellychurchill) [inline src/content/LandingPage/LandingPage.js]: This row should come after row 2

File: src/content/LandingPage/LandingPage.js [hunk 2]
```diff
@@ -1,7 +1,104 @@
...
+    tabIndex: 0,
+  },
+};
 
 const LandingPage = () => {
-  return <div>LANDING PAGE</div>;
+  return (
+    <div className="bx--grid bx--grid--full-width landing-page">
+      <div className="bx--row landing-page__banner">
+        <div className="bx--row landing-page__banner">
```
Comment (kellychurchill) [inline src/content/LandingPage/LandingPage.js]: Duplication of `landing-page__banner` class is causing extra padding for this section.

Comment (Taciones) [reply src/content/LandingPage/LandingPage.js]: True... Looks better now. 
Thanks!

**Comment by kellychurchill** [comment]
👍 Looks good - 🎉 You are one step closer to completion of the Carbon React tutorial and earning the digital credential.

**Pull request closed without merging**